src/js/utils/RouterUtil.js

```diff
@@ -17,28 +17,63 @@ function findRedirect(queryString) {
 }
 
 const RouterUtil = {
+  /**
+   * Parse the url and find the query string (?)
+   * before or after the #
+   *
+   * @returns {Boolean|Object} false or query string object
+   */
+  getQueryStringInUrl() {
```

Comment:
Please make sure we write tests for these methods in RouterUtil. Looks like we could do with more coverage 👍